Nodejs/Product/TestAdapterNetStandard/NetCoreTestDiscoverer.cs

```diff
@@ -147,10 +119,76 @@ void ProcessFiles(string fileAbsolutePath)
             }
         }
 
-        private void DiscoverTests(HashSet<string> fileList, TestFramework testFramework, ITestCaseDiscoverySink discoverySink, IMessageLogger logger, string nodeExePath, string projectFullPath)
+        private void DiscoverTests(Dictionary<string, HashSet<string>> testItems, FrameworkDiscoverer frameworkDiscoverer, ITestCaseDiscoverySink discoverySink, IMessageLogger logger, string nodeExePath, string projectFullPath)
         {
-            var discoverWorker = new TestDiscovererWorker(projectFullPath, NodejsConstants.ExecutorUri, nodeExePath);
-            discoverWorker.DiscoverTests(fileList, testFramework, logger, discoverySink);
+            foreach (var testFx in testItems.Keys)
+            {
+                var testFramework = frameworkDiscoverer.GetFramework(testFx);
+                if (testFramework == null)
+                {
+                    logger.SendMessage(TestMessageLevel.Warning, $"Ignoring unsupported test framework '{testFx}'.");
+                    continue;
+                }
+
+                var fileList = testItems[testFx];
+
+                var discoverWorker = new TestDiscovererWorker(projectFullPath, NodejsConstants.ExecutorUri, nodeExePath);
+                discoverWorker.DiscoverTests(fileList, testFramework, logger, discoverySink);
+            }
+        }
+
+        private Dictionary<string, HashSet<string>> GetConfigItems(string projectRoot)
+        {
+            var files = Directory.EnumerateFiles(projectRoot, "angular.json", SearchOption.AllDirectories)
+                .Where(x => !x.Contains("\\node_modules\\"));
+            if (files.Any())
+            {
+                return new Dictionary<string, HashSet<string>>()
+                {
+                    {
+                        TestFrameworkDirectories.AngularFrameworkName,
+                        new HashSet<string>(files)
+                    }
+                };
+            }
+
+            return default;
+        }
+
+        private Dictionary<string, HashSet<string>> GetTestItems(string projectRoot, string testFramework, string outDir)
```

Comment:
This is a really expensive way to represent the data being returned, but maybe it will make more sense once I've read the rest of the file.

Reply:
I'm still not getting it - it looks like the dictionary will always have either 0 or 1 elements.

Reply:
This is a copy paste from the other tests adapters (where they might contain different test frameworks). Fixed.